script/buildSQLGeom.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env python
+
+"""
+This module will accept an input text file with (x,y,z) coordinates in a CSV-format list
```

Comment:
Please try usgs query service http://nationalmap.gov/epqs/ for US locations. It's quite easy to figure out a http get request to this service using code.